src/utils/yup/index.ts

```diff
@@ -135,3 +154,78 @@ function createUnknownError(createError: TestContext['createError']) {
     message: formatMessage({ id: 'error.unknown' }),
   });
 }
+
+/**
+ * Debounce yup validation test function, inspired by lodash's debounce function.
```

Comment:
Added a custom yup validation debounce function. Lodash's debounce doesn't work for this use case as explained below.